### PR TITLE
Add link on the website for reporting a security vulnerability

### LINKED ISSUE #4515: Add a link on "how to report a security vulnerability" on the website
Every Eclipse IoT project is expected to have a link on "how to report a security vulnerability" on its website, see https://wiki.eclipse.org/IoT/PMC#Releases.

We therefore need to add such.

### PR DESCRIPTION
**Pull request by triller-telekom**
Fixes #4515

Signed-off-by: Stefan Triller <stefan.triller@telekom.de>

### CODE REVIEW

**Comment by triller-telekom** [comment]
Just a comment on this: I deliberately used the word "security" on the first site because I know that security research search for this term if they would like to report a vulnerability.

Feel free to integrate your thoughts if you think my proposal should be improved.

File: docs/index.html [hunk 1]
```diff
@@ -244,13 +244,15 @@ <h1>Latest Tweets</h1>
 			<h1>Join the Community!</h1>
 			<p>Eclipse SmartHome™ is an open-source project that is driven by
 				its community. Involve yourself now by joining discussions,
-				by reporting issues and by contributing code!</p>
+				by reporting (<a href="https://eclipse.org/security">security related</a>) issues and by contributing code!</p>
```
Comment (kaikreuzer) [inline docs/index.html]: Now "normal" issues are somehow gone. I'd reformulate it to:

> by reporting issues or vulnerabilities and by contributing code! 

(with the link on "vulnerabilities")

File: docs/index.html [hunk 2]
```diff
@@ -244,13 +244,15 @@ <h1>Latest Tweets</h1>
...
 			<a href="https://www.eclipse.org/forums/index.php/f/271/" class="btn waves-effect waves-light green darken-1"
 				type="submit" name="action">Discussion Forum</a>
 			<a href="https://bugs.eclipse.org/bugs/buglist.cgi?list_id=12033781&product=SmartHome&query_format=advanced&resolution=---" class="btn waves-effect waves-light green darken-1"
 				type="submit" name="action">Issue Tracker</a>
 			<a href="https://github.com/eclipse/smarthome" class="btn waves-effect waves-light green darken-1"
 				type="submit" name="action">Git Repository</a>
+			<a href="https://eclipse.org/security" class="btn waves-effect waves-light green darken-1"
```
Comment (kaikreuzer) [inline docs/index.html]: I would not add such a huge button for it, having the link in the text above is imho enough,

File: docs/index.html (hunk 1 above)
Comment (triller-telekom) [reply docs/index.html]: Why are "normal" issues gone now? I have put "security related" in braces before "issues".

And I would really like to see the term "security" somehow, as I have mentioned that security researchers do search for that.

File: docs/index.html (hunk 2 above)
Comment (triller-telekom) [reply docs/index.html]: Ok, but then please see my comment above.

**Pull request merged**